src/promptflow-devkit/pyproject.toml

```diff
@@ -67,6 +67,7 @@ pyinstaller = ">=5.13.2"
 streamlit = ">=1.26.0"
 streamlit-quill = "<0.1.0"
 bs4 = "*"
+tomli = "*"
```

Comment:
add comments when is this used. should be an extra?

Reply:
Use in build executable.